pdf-workitem/pom.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm.contrib</groupId>
+    <artifactId>workitems</artifactId>
+    <version>7.12.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pdf-workitem</artifactId>
+  <name>PDF</name>
+  <description>Create PDF from Thymeleaf template</description>
+  <organization>
+    <name>jBPM dev team</name>
+  </organization>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.thymeleaf</groupId>
+      <artifactId>thymeleaf</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>javassist</groupId>
+          <artifactId>javassist</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.xhtmlrenderer</groupId>
+      <artifactId>flying-saucer-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.xhtmlrenderer</groupId>
+      <artifactId>flying-saucer-pdf-itext5</artifactId>
```

Comment:
So this will bring in itext? Wondering if that won’t cause conflicts...

Reply:
I created a demo process which creates a pdf and uploads it on dropbox using that workitem. with the depends as they are in the module pom it does not cause errors in the workbench

Reply:
perfect, could you make a reference to that sample somewhere so people can get easy started with it... maybe we should have a new organisation in GitHub for all these examples

Reply:
i was thinking of adding these examples as part of the repository itself (another tab like examples that users can import) wdyt?

Reply:
+1